README.md

```diff
@@ -0,0 +1,38 @@
+# hello_world_flask
+Project to learn and try out Flask and rq
+
+Clone the repo on your machine. 
+
+You need 
+- [Flask](https://flask.palletsprojects.com/en/1.0.x/installation/#installation)
+- [Redis](https://redis.io/download)
+- [rq](https://python-rq.org/)
+- [PopPUNK](https://github.com/johnlees/PopPUNK)
+
+installed. 
+
+1. Start the Redis server with `src/redis-server` from the redis installation folder
```

Comment:
Is this a script you have written? I can't see it in this PR

Reply:
This is an executable that comes with the Redis installation. Do the instructions on how to start the Redis server need clarification?

Reply:
No I was just being lazy and hadn't installed that! You could try using the redis docker image here too https://hub.docker.com/_/redis